.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Why is this necessary in the repo? Historically we don't add support for IDE artifacts.

Reply:
An accidental inclusion. It's removed. Thanks for catching it.